removed-packages/core/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@emotion/core",
+  "version": "11.0.0-next.9",
+  "main": "dist/core.cjs.js",
+  "module": "dist/core.esm.js",
+  "browser": {
```

Comment:
nit-picking, but we dont need all of the extra alias fields etc here, just `"main"` is sufficient